src/ResourceManager/Insights/Commands.Insights.Test/Utilities.cs

```diff
@@ -246,7 +244,7 @@ public static void ExecuteVerifications(LogsCmdletBase cmdlet, Mock<IActivityLog
             VerifyFilterIsUsable(filter: filter);
             VerifyStartDateInFilter(filter: filter, startDate: startDate);
             VerifyConditionInFilter(filter: filter, field: requiredFieldName, value: requiredFieldValue);
-            VerifyContinuationToken(nextLink: nextLink);
+            //// VerifyContinuationToken(nextLink: nextLink);
```

Comment:
@pkamath13 @gucalder the rest of these commented lines should be removed from this file 😀

Reply:
Done